build.sbt

```diff
@@ -177,13 +178,19 @@ lazy val play25ConductRClientLib = project
   .enablePlugins(CrossPerProjectPlugin)
 
 // Lagom version 1
-lazy val lagom1ConductRBundleLib = project
-  .in(file("lagom1-conductr-bundle-lib"))
+lazy val lagom1ConductRBundleLibJavadsl = project
```

Comment:
Completely up to you.  I named it with Javadsl at the end since this is the convention we've used in Lagom, but happy to change it to whatever you think it should be.